projects/packages/analyzer/scripts/jp-warnings-job.php

```diff
@@ -4,18 +4,10 @@
 
 // ini_set( 'memory_limit', '512M' );
 
-$excludes = array( '.git', 'vendor', 'tests', 'docker', 'bin', 'scss', 'images', 'docs', 'languages', 'node_modules' );
-
-// $jetpack_new_path = '/Users/brbrr/Developer/a8c/jetpack/projects/plugins/jetpack';
-// $jetpack_old_path = '/Users/brbrr/Developer/a8c/jetpack/projects/packages/analyzer/jetpack-production';
 $folder_name = $argv[1];
+$excludes = array( '.git', 'vendor', 'tests', 'docker', 'bin', 'scss', 'images', 'docs', 'languages', 'node_modules' );
 
-$slurper_path = dirname( __DIR__ ) . '/slurper/plugins';
-
-// $jetpack_new_path = '/Users/brbrr/Developer/a8c/jetpack/projects/plugins/jetpack';
-$jetpack_new_path = '/var/www/html/wp-content/plugins/jetpack';
-$jetpack_old_path = $slurper_path . '/jetpack-production';
-$differences      = Automattic\Jetpack\Analyzer\Scripts::get_differences( $jetpack_new_path, $jetpack_old_path );
+$differences      = Automattic\Jetpack\Analyzer\Scripts::get_differences( '', '' );
```

Comment:
Empty string for both paths? How does that work?

Reply:
That's probably an area for improvement. `get_differences` checks if a JSON file exists before scanning provided paths. Since this script is called by the parent `jp-analyze-parallel.php` after the differences were generated, we are sure differences would be loaded from the file.

Reply:
Ah, I see. Maybe a `load_differences()` that just loads the `diff.json`?

Reply:
Added in 706d7d9